app/containers/UserAuthNavLinks/index.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Pls make UserProfileNav a separate container. In the future it will become more complex. It will need to load user's avatar, potentially status and some other details.
In that case UserAuthNavLinksComponent is not needed (Component suffix is bad). UserAuthNavLinks will be just links.
The logic deciding what to show, UserAuthNavLinks or UserProfileNav moves level above, to Header (guess Header will need to be converted to container in that case).
NOTE: we can keep it as it is for now. That can be refactored when we work on User Profile navigation in the header.